src/main/kotlin/io/stud/forest/book_a_cinema_ticket_app/domain/TimeInterval.kt

```diff
@@ -0,0 +1,16 @@
+package io.stud.forest.book_a_cinema_ticket_app.domain
+
+import java.time.LocalTime
+
+data class Hour(
+        val value: Int
+) {
+    init {
+        require(value in 1..23) { "hour can take values from 1 to 23" }
```

Comment:
A 0 nie może? Poza tym to sprawdzasz tutaj tworząc co i rusz nową listę/sekwencję. Może wystarczyłoby >0 && <24?

Reply:
Dokładnie tak.